spec/integration/output/slate/index.html.md

```diff
@@ -223,3 +223,31 @@ Parameter | Type | Description
 --------- | ------- | -----------
 id | integer | The id of the place
 name | string | The place's name
+
+## Listing all places with a modified response bod,
+
+
+
```

Comment:
All the whitespace comments are for the generated files that I don't control the format of. This one is for use with [slate](https://github.com/lord/slate).